include/Surelog/Common/SymbolId.h

```diff
@@ -22,33 +23,28 @@ namespace SURELOG {
  *
  * Used to uniquely represent a string in SymbolTable. SymbolId can (and
  * should) be resolved only with the SymbolTable that it was generated with.
- *
+ * 
```

Comment:
Trailing whitespace (if you configure your editor to use the `.editorconfig`, this can be automatic removed)